al2023-arm64.sha256

```diff
@@ -0,0 +1 @@
+375e917134782efb468c2c4f9f9a00d299c0a1140476d9471a0a9615dbf63158
```

Comment:
Doesn't appear to be used.

Reply:
File has been renamed to version.sha256 and is copied into the AL container.